tests/performance/cases/basic_coresets.py

```diff
@@ -0,0 +1,107 @@
+"""
+Simple coreset performance tests.
+
+These were adapted from the `test_herding_basic` integration test.
+"""
+
+import functools
+
+import numpy as np
+from jax import numpy as jnp
+from jax import random
+from sklearn.datasets import make_blobs
+
+from coreax import (
+    Data,
+    KernelDensityMatching,
+    PCIMQKernel,
+    SquaredExponentialKernel,
+    SteinKernel,
+)
+from coreax.kernels import median_heuristic
+from coreax.solvers import KernelHerding, RPCholesky, SteinThinning
+from coreax.util import JITCompilableFunction
+
+CORESET_SIZE = 100
```

Comment:
I think I would be in favour of putting `num_data_points` and `num_features` as global constants here as well, as this will have an effect on runtime